FusionIIIT/applications/central_mess/views.py

```diff
@@ -394,6 +391,22 @@ def mess(request):
         }
         return render(request, 'messModule/mess.html', context)
 
+@login_required
+@transaction.atomic
+@csrf_exempt
+def mess_info(request):
+    if (request.method == "POST"):
+        user_id = request.user
+        student_id = Student.objects.select_related('id','id__user','id__department').get(id__id=user_id)
+        mess_option = request.POST.get('mess_option')

```

Comment:
try using form.is_valid() and form.cleaned_data instead of retrieving data from request.POST. refer [here](https://stackoverflow.com/questions/10877580/should-i-use-the-text-from-request-post-or-form-cleaned-data) for reason and refer [doc](https://docs.djangoproject.com/en/4.0/topics/forms/#the-view) for knowing more about working with forms

Reply:
done